### DISCUSSION TITLE
milvus 的 expr 过滤的时候只能用 运算符右侧只能是常量吗

### DISCUSSION THREAD

**Discussion by YCYyuxiangbuyu** [milvus-io/milvus#35175] [Q&A and General discussion]
milvus 的 expr 过滤的时候只能用 运算符右侧只能是常量吗

**Comment by yhmo** [comment]
Assume the collection has these fields:
"id" Int64
"doc_id" Int64
"content" Varchar
"title" Varchar

| expr | legal? |
|:--------------:|:----------------:|
|expr = "id > doc_id"|yes|
|expr = "id > doc_id + 1"|no|
|expr = "doc_id == id"|yes|
|expr = "doc_id == id + 1"|no|
|expr = "content == title"|yes|
